modules/elements-color-picker/assets/js/editor/commands/show-swatches.js

```diff
@@ -225,6 +237,81 @@ export class ShowSwatches extends CommandBase {
 		this.backgroundImages = [];
 	}
 
+	/**
+	 * Add the color swatches to a picker container.
+	 *
+	 * @param {HTMLElement} picker - Picker HTML element to append the swatches to.
+	 *
+	 * @returns {void}
+	 */
+	addColorSwatches( picker ) {
```

Comment:
Don't bother reviewing this function. It was just extracted as-is from `initSwatch()`.